pipelines.adoc

```diff
@@ -1,266 +1,189 @@
-
-Note: Currently the featured collections share the same tasks &
-pipelines and the collections build process will copy the files from
-https://github.com/kabanero-io/collections/tree/master/incubator/common/pipelines/default
-to /pipelines during build. If you are building a new collection, you
-can leverage the existing pipelines as is if it satisfies your
-requirements to build and deploy. They will automatically be pulled into
-your collection when you build your collections repo.
-
-If you want to change the tasks or pipelines and want it to apply to all
-the collections, you can updates the files under
-https://github.com/kabanero-io/collections/tree/master/incubator/common/pipelines/default.
-If you would like to make changes to tasks or pipelines specific to a
-collection, you can create the files under /pipelines in you collections
+---
+permalink: /guides/pipelines-getting-started/
+---
+:page-layout: guide
+:page-duration: 30 minutes
+:page-releasedate: 2019-10-17
+:page-description: Explore how to use Kabanero Pipelines to build and manage Kabanero collections.
+:page-tags: ['Collection']
+:page-guide-category: pipelines
+
+== Working with Kabanero Pipelines
+
+Kabanero uses link:https://github.com/tektoncd/pipeline/tree/master/docs#usage[Tekton Pipelines] to illustrate a CI/CD workflow. Each of the featured link:https://github.com/kabanero-io/collections[Kabanero collections] contains a default pipeline that builds the collection, publishes the image to a docker repo, and then deploys the image to the Kubernetes cluster. You can add more tasks and pipelines to the collections. All the tasks and pipelines are activated by the link:https://github.com/kabanero-io/kabanero-operator[Kabanero operator].
+
+To learn more about Tekton Pipelines and creating new tasks, see link:https://github.com/tektoncd/pipeline/blob/master/docs/tutorial.md[the Tekton Pipeline tutorial].
+
+== Kabanero Tasks and Pipelines
+
+Each Kabanero collection contains default tasks & pipelines that are created when the collection is built. Currently,  all the featured collections share tasks & pipelines. The collections build process copies the files from link:https://github.com/kabanero-io/collections/tree/master/incubator/common/pipelines/default[the collections repo] to your `/pipelines` directory.
+
+If you are building a new collection, you can use these existing pipelines as they are if that satisfies your requirements to build and deploy. They are automatically pulled into your collection when you build your collections repo. If you want to change the tasks or pipelines, you can choose to have your changes apply either to all collections or a specific collection.  To apply your changes to all the collections,  update the files in the `incubator/common/pipelines/default` in your collections repo. If you want to update tasks or pipelines in a particular collection, create the files in the `/pipelines` directory in your collections
 repo.
```

Comment:
can we change this to "create the files in the pipelines directory under the specific collection in your collections repo.".